MobileCenter/MobileCenterTests/Util/MSTestFrameworks.h

```diff
@@ -1,4 +1,11 @@
 #import <Foundation/Foundation.h>
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
+#pragma clang diagnostic ignored "-Wdocumentation-deprecated-sync"
+#pragma clang diagnostic ignored "-Wdocumentation-unknown-command"
+#pragma clang diagnostic ignored "-Wobjc-interface-ivars"
```

Comment:
It reduces warnings from third-party libraries. Currently we have more than 100 warnings from it. Because this isn't related to our code we can ignore they.